app.shazel.pcheetah.json

```diff
@@ -0,0 +1,126 @@
+{
+ "_cli": '
+flatpak-builder --repo=_repo _build _mani
+         ',
+ "app-id":  "app.shazel.pcheetah",
```

Comment:
the domaun `shazel.app` doesn't exist. This app id doesn't meet the requirement.